src/Tinker.jl

```diff
@@ -498,12 +496,49 @@ push!(zoom_ctrl["enabled"], false)
         drawrecthandle(ctx, rh, colorant"blue", colorant"white")
     end
 
-showall(win);
+    showall(win);
 
-append!(c.preserved, [zoom_ctrl, pandrag, rectselect])
+    append!(c.preserved, [zoom_ctrl, pandrag, rectselect])
 
+    imagectx = Dict("Image"=>image, "Canvas"=>c, "ZoomRegion"=>zr,"Rectangle"=>
+                    rect, "MouseActions"=>[pandrag, zoom_ctrl, rectselect])
+    push!(img_ctxs, push!(value(img_ctxs), imagectx))
+    return imagectx
 end;
 
 init_gui(file::AbstractString) = init_gui(load(file); name=file)
 
+active_context = map(img_ctxs) do ic # signal dependent on img_ctxs
+    if isempty(ic)
+        # placeholder value of appropriate type
+        init_gui(testimage("lighthouse")) # find a better placeholder
```

Comment:
What's the idea here? Is it that the gui doesn't start out empty? I think a `TestImages` dependency is a bit much.

How about something like 

```julia
img = [RGB(r,0.5,b) for r in linspace(0,1,200), b in linspace(0,1,300)]
```

![testimage](https://user-images.githubusercontent.com/10854026/27979314-54dfe682-6375-11e7-8367-fde794300629.png)

What do you mean with "appropriate type" because images will in general have very different eltypes.